paper-world-defaults.yml

```diff
@@ -295,12 +295,140 @@ spawn:
 tick-rates:
   behavior:
     villager:
-      validatenearbypoi: -1
+          validatenearbypoi: 60
+          acquirepoi: 120
   container-update: 1
   grass-spread: 4
   mob-spawner: 1
   sensor:
+    allay:
+      hurtbysensor: 40
+      nearestitemsensor: 20
+      nearestlivingentitysensor: 20
+      playersensor: 20
+    armor_stand:
+    axolotl:
+      adultsensor: 40
+      axolotlattackablessensor: 40
+      hurtbysensor: 40
+      nearestlivingentitysensor: 40
+      temptingsensor: 40
+    bat:
```

Comment:
For the mobs that don't have any sensors, I'd remove them to clear up some space